client/src/components/layout/Header.js

```diff
@@ -19,7 +19,7 @@ function Header() {
             <Navbar.Brand><Link className="header-brand" to="/dashboard">Researchify</Link></Navbar.Brand>
             <Nav className="mr-auto">
                 <Nav.Link><Link className="header-link" to="/dashboard/profile">Profile</Link></Nav.Link>
-                <Nav.Link><Link className="header-link" to="/publications/team/606bb59c22201f529db920c9">Publications </Link></Nav.Link>
+                <Nav.Link><Link className="header-link" to="/publications/team">Publications </Link></Nav.Link>
```

Comment:
@Researchify/logiciel  has removed the links from Header.js in PR#78 and put them in Sidebar.js, but still waiting to merge though, do we need to change them in Sidebar.js as well?

Reply:
Can you guys merge first assuming you're already ready? We can deal with the conflict

Reply:
Sorry that PR#78 is blocked, I guess Scholar can merge first and we can do the patch on Logiciel later.